anyway/infographics_utils.py

```diff
@@ -275,14 +275,29 @@ def get_most_severe_accidents(table_obj, filters, start_time, end_time, limit=10
         entities = (
             "longitude",
             "latitude",
-            "accident_severity_hebrew",
+            "accident_severity",
             "accident_timestamp",
-            "accident_type_hebrew",
+            "accident_type",
         )
         return get_most_severe_accidents_with_entities(
             table_obj, filters, entities, start_time, end_time, limit
         )
 
+    @staticmethod
+    def localize_items(request_params: RequestParams, items: Dict) -> Dict:
+        for item in items["data"]["items"]:
+            # noinspection PyBroadException
```

Comment:
That's not the way to fix it. . .
Read my comment about the 4 options to discover the exception types